islandora-indexing-fcrepo/src/main/java/ca/islandora/alpaca/indexing/fcrepo/FcrepoIndexer.java

```diff
@@ -225,7 +226,18 @@ public void configure() {
                 .setBody(simple("${null}"))
 
                 // Pass it to milliner.
-                .toD(getMillinerBaseUrl() + "media/${exchangeProperty.sourceField}?connectionClose=true");
+                .toD(getMillinerBaseUrl() + "media/${exchangeProperty.sourceField}?connectionClose=true")
+                .choice()
+                        .when()
+                        .simple("${exchangeProperty.event.object.isNewVersion}")
+                                .setHeader("Content-Location", simple(
+                                        "${exchangeProperty.jsonUrl}"))
+
+                                //pass it to milliner
+                                .toD(
+                                        getMillinerBaseUrl() +
+                                        "media/${exchangeProperty.sourceField}/version?connectionClose=true"
+                                    ).endChoice();
```

Comment:
Funky indendation here

Reply:
do you mean because i put 'getMillinerBaseUrl....' on the next line?
i'm pretty sure it was throwing an error about the line being too long.

Reply:
I was expecting the ending parenthesis before `.endChoice` to be aligned with `.toD`. Maybe that's just me.

No biggie though.  It passes codestyle checks so I'm fine pulling in this now.

Reply:
ok i can fix it